webpack/envs/commonLoaders.js

```diff
@@ -21,6 +21,26 @@ export const babelLoader = {
   ],
 }
 
+export const babelLoaderWithLodash = {
+  exclude: /(node_modules)/,
+  include: path.resolve(basePath, "src"),
+  test: /(\.(js|ts)x?$)/,
+  use: [
+    {
+      loader: "babel-loader",
+      options: {
+        cacheDirectory:
+          !env.onCi && path.join(basePath, ".cache", "babel/force"),
+        plugins: [
+          "lodash",
```

Comment:
Why are we now [duplicating config](https://github.com/artsy/force/blob/master/babel.config.js#L43)?

Reply:
In response to the original, edit.

"Why are we now duplicating config?"

It's not duplicated when it's different.

In response to the future edits, naming is hard. It's tailored for lodash, so the name reflects that. Helpful suggestions welcomed.

"It seems a bit specific when folks looking at config know the usual suspects."

I'm not following this? Pretty sure only us two touch these files. Mostly myself.

Reply:
> In response to the original, edit.

Everything does not need to be a defensive fight. The edit button is there for a reason and you don't need to pedantically comment on every minor revision that I make on a PR comment that I'm using to craft a thought. I asked a simple question about there seemingly being two lodash babel plugins used in two separate spots, and then was wondering about naming. Chill.